docs/sources/visualizations/histogram-draft.md

```diff
@@ -0,0 +1,161 @@
++++
+title = "Histogram"
+description = "Histogram visualization"
+keywords = ["grafana", "docs", "bar chart", "panel", "barchart"]
+aliases =["/docs/grafana/latest/features/panels/histogram/", "/docs/grafana/latest/panels/visualizations/histogram/"]
+weight = 605
++++
+# Visualize Prometheus histogram in Grafana
+Assuming users have a basic understanding of [Prometheus](https://grafana.com/oss/prometheus/) and Grafana, this documentation will look at histograms from the perspective of [Grafana 7.0](https://grafana.com/blog/2020/05/18/grafana-v7.0-released-new-plugin-architecture-visualizations-transformations-native-trace-support-and-more/).
```

Comment:
I suggest you use rel refs...if I have time I can try to find the paths, but you can look in the Cloud topics in the `grafana/website` repo for example formatting.

Reply:
Yes, this is really a WIP, not ready for review yet.